shepherd.js/src/step.ts

```diff
@@ -184,6 +189,18 @@ export interface StepOptions {
    * ```
    */
   when?: StepOptionsWhen;
+
+  /**
+   * An array of extra element selectors to highlight when the overlay is shown
+   * The tooltip won't be fixed to these elements, but they will be highlighted
+   * just like the `attachTo` element.
+   * ```js
+   * const step = new Step(tour, {
+   *   extraHighlights: [ '.pricing', '#docs' ],
+   *   ...moreOptions
+   * });
+   */
+  extraHighlights?: ReadonlyArray<string>;
```

Comment:
I think these are in alphabetical order. Would you mind adjusting placement of this please?

Reply:
Oh I didn't notice that! Fixed, @RobbieTheWagner